### DISCUSSION TITLE
Example is not running inside this repo.

### DISCUSSION THREAD

**Discussion by bhavinatharva** [shogo4405/HaishinKit.kt#208] [Q&A] [answered]
I am getting this error 

```
#include <vulkan/vulkan.hpp>
```

In file included from /Users/bhavinp/Workspace/AndroidWorkspace/HaishinKit.kt/vulkan/src/main/cpp/Graphics/Kernel.cpp:1:
/Users/bhavinp/Workspace/AndroidWorkspace/HaishinKit.kt/vulkan/src/main/cpp/Graphics/Kernel.h:10:10: fatal error: 'vulkan/vulkan.hpp' file not found
#include <vulkan/vulkan.hpp>
         ^~~~~~~~~~~~~~~~~~~

**Comment by shogo4405** [comment, marked as answer]
It's possible that the 'git submodule' command isn't working properly. Did you execute this command?
```
git clone https://github.com/shogo4405/HaishinKit.kt.git
cd HaishinKit.kt
git submodule update --init
^~~~~~~~~~~~~~~~~~~~~~

# Open [Android Studio] -> [Open] ...
```

**Reply by bhavinatharva** [reply]
No yet, do I need to run that command?

**Reply by shogo4405** [reply]
Sure.

**Reply by bhavinatharva** [reply]
Thank you, it is running, but I am not getting camera view , is any need require for that?
![Screenshot_1704267282](https://github.com/shogo4405/HaishinKit.kt/assets/84075780/788415b5-96e6-4320-926d-88292448d17d)

**Reply by bhavinatharva** [reply]
I am getting this error

```
android.hardware.camera2.CameraAccessException: CAMERA_ERROR (3): The camera device has encountered a serious error
```

**Reply by shogo4405** [reply]
There might be compatibility issues with Vulkan causing the error. Please comment out this line here.
https://github.com/shogo4405/HaishinKit.kt/blob/main/app/src/main/java/com/haishinkit/app/MainActivity.kt#L18-L20

**Reply by bhavinatharva** [reply]
ok let me check
